vips/vip-501/bscmainnet.ts

```diff
@@ -0,0 +1,346 @@
+        params: [
+          [
+            weETH.address,
+            [WEETH_ORACLE, ethers.constants.AddressZero, ethers.constants.AddressZero],
+            [true, false, false],
+            false,
+          ],
+        ],
+        dstChainId: LzChainId.unichainmainnet,
+      },
+
+      // Market configurations
+      {
+        target: weETHMarket.vToken.address,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["86400"],
+        dstChainId: LzChainId.unichainmainnet,
+      },
+      {
+        target: unichainmainnet.VTREASURY,
+        signature: "withdrawTreasuryToken(address,uint256,address)",
+        params: [
+          weETHMarket.vToken.underlying.address,
+          weETHMarket.initialSupply.amount,
+          unichainmainnet.NORMAL_TIMELOCK,
+        ],
+        dstChainId: LzChainId.unichainmainnet,
+      },
+      {
+        target: weETH.address,
+        signature: "approve(address,uint256)",
+        params: [unichainmainnet.POOL_REGISTRY, 0],
+        dstChainId: LzChainId.unichainmainnet,
+      },
+      {
+        target: weETH.address,
+        signature: "approve(address,uint256)",
+        params: [unichainmainnet.POOL_REGISTRY, weETHMarket.initialSupply.amount],
+        dstChainId: LzChainId.unichainmainnet,
+      },
+      {
+        target: unichainmainnet.POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            weETHMarket.vToken.address,
+            weETHMarket.riskParameters.collateralFactor, // CF
+            weETHMarket.riskParameters.liquidationThreshold, // LT
+            weETHMarket.initialSupply.amount, // initial supply
+            unichainmainnet.NORMAL_TIMELOCK, // vToken receiver
+            weETHMarket.riskParameters.supplyCap, // supply cap
+            weETHMarket.riskParameters.borrowCap, // borrow cap
+          ],
+        ],
+        dstChainId: LzChainId.unichainmainnet,
+      },
+      {
+        target: weETHMarket.vToken.address,
+        signature: "transfer(address,uint256)",
+        params: [ethers.constants.AddressZero, weETHMarket.initialSupply.vTokensToBurn],
+        dstChainId: LzChainId.unichainmainnet,
+      },
+      (() => {
+        const vTokensMinted = convertAmountToVTokens(weETHMarket.initialSupply.amount, weETHMarket.vToken.exchangeRate);
+        const vTokensRemaining = vTokensMinted.sub(weETHMarket.initialSupply.vTokensToBurn);
+        return {
+          target: weETHMarket.vToken.address,
+          signature: "transfer(address,uint256)",
+          params: [weETHMarket.initialSupply.vTokenReceiver, vTokensRemaining],
+          dstChainId: LzChainId.unichainmainnet,
+        };
+      })(),
+      {
+        target: weETHMarket.vToken.address,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [weETHMarket.riskParameters.protocolSeizeShare],
+        dstChainId: LzChainId.unichainmainnet,
+      },
```

Comment:
Unnecessary, because the current value is already correct

Reply:
https://github.com/VenusProtocol/vips/pull/561/commits/a307d272d9a8bb706bf64850057229e97c421d4b